package.json

```diff
@@ -2,19 +2,21 @@
   "name": "roomies-project",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:8080",
```

Comment:
Is this a setting that should be committed? Or is this something used for local testing?

Reply:
It's for local testing. I've removed it